index.html

```diff
@@ -0,0 +1,157 @@
+<!DOCTYPE html>
+
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="./style.css" type="text/css" />
+    <link rel="shortcut icon" type="image/png"
+      href="image/logo.png">
+   
+      <title>Movie Night</title>
+
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Archivo:wght@300&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Actor&family=Klee+One:wght@600&family=Pangolin&display=swap" rel="stylesheet">      
+    </head><body>
+    
+
+    
+
+    <div class="container-nav">
+
+      <div class="nawbar">
+        
+       
+<div class="navbar-items">
+
+
+          <img id="nav-img" src="image/logo.png"/>
+          <h1 id="nav-text">Movie Night</h1>
+
+    
+        
+          <a id="nav-button-1" href="#bignewz" class="nawbar-btn">Veckans Nyhet</a>
+          <a id="nav-button-2" href="#regular-news-section" class="nawbar-btn">Tidigare tips</a>
+          <a id="nav-button-3" href="https://www.aftonbladet.se/tipsa" target="_blank" class="nawbar-btn">Kontakta oss</a>
+          <a id="nav-button-4" href="https://www.aftonbladet.se/nojesbladet/tv" target="_blank" class="nawbar-btn">Utforska</a>
+        </div>
+      </div>
+      </div>
+    
+    
+      <div class="container-header">
+
+      <header>
```

Comment:
Just thinking of html semantics - perhaps consider to rename the `<header>` tag to something else, for example `<section>`. Or is there a reason you call this a header?